solver.rb

```diff
@@ -0,0 +1,20 @@
+class Solver
+  def factorial(num)
+    raise 'The number should be greater than zero' if num.negative?
+    return 1 if num.zero?
+
+    num * factorial(num - 1)
+  end
+
+  def reverse(word)
+    word.chars.reverse.join
+  end
+
+  def fizzbuzz(num)
+    return 'fizzbuzz' if (num % 3).zero? && (num % 5).zero?
+    return 'fizz' if (num % 3).zero?
+    return 'buzz' if (num % 5).zero?
+
+    num.to_s
+  end
+end
```

Comment:
- [ ] Clean code 💯

Reply:
Your code is clean and easy to understand. Keep it up 💪

Reply:
Thank you so much @julie-ify for your feedback.